tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -402,6 +402,13 @@ test_i2c_vl53l0x: vl53l0x@40 {
 	xshut-gpios = <&test_gpio 0 0>;
 };
 
+test_i2c_vl53l1x: vl53l1x@29 {
+	compatible = "st,vl53l1x";
+	reg = <0x29>;
+	int-gpios = <&test_gpio 0 0>;
+	xshut-gpios = <&test_gpio 0 0>;
+};
```

Comment:
Please put device at the end of the list with a unique reg address.
I guess this will produce a warning as 29 is already used.

Reply:
yes, sorry, I didn't notice how the devices were structured like that. I defaulted to sticking this TOF sensor next to the only other one.

@erwango  Note that at the end of this file, there is a device (`test_i2c_as5600`) with a reg that doesn't follow the convention and has a duplicate reg value of 54. It's probably a typo and should be 65. Should I change it here in my PR and stick my device below with reg 65? See below (before and proposed change).

```
...
test_i2c_icp10125: icp10125@63 {
	compatible = "invensense,icp10125";
	reg = <0x63>;
	temperature-measurement-mode = "normal";
	pressure-measurement-mode = "normal";
};

test_i2c_as5600: as5600@54 {
	compatible = "ams,as5600";
	reg = <0x54>;
};
EOF
```
changed to 
```
test_i2c_icp10125: icp10125@63 {
	compatible = "invensense,icp10125";
	reg = <0x63>;
	temperature-measurement-mode = "normal";
	pressure-measurement-mode = "normal";
};

test_i2c_as5600: as5600@64 {
	compatible = "ams,as5600";
	reg = <0x64>;
};

test_i2c_vl53l1x: vl53l1x@65 {
	compatible = "st,vl53l1x";
	reg = <0x65>;
	int-gpios = <&test_gpio 0 0>;
	xshut-gpios = <&test_gpio 0 0>;
};
EOF
```

Reply:
This is what I'll do and can revert if necessary